README.md

```diff
@@ -1,14 +1,127 @@
-# ludex-sdk-js [![Build](https://github.com/Ludex-Labs/ludex-sdk-js/actions/workflows/build.yml/badge.svg)](https://github.com/Ludex-Labs/ludex-sdk-js/actions/workflows/build.yml)
+## The Official Javascript & Typescript SDK for Ludex API
 
-[Ludex](https://ludex.gg/) provides video game developers an easy way to implement smart contracts into games with no blockchain knowledge required. This is the Javascript SDK for interacting with our API and smart contracts.
+[![Build](https://github.com/Ludex-Labs/ludex-sdk-js/actions/workflows/build.yml/badge.svg)](https://github.com/Ludex-Labs/ludex-sdk-js/actions/workflows/build.yml)
 
-## Installation
+### About
 
-```sh
-npm i @ludex-labs/ludex-sdk-js
-```
+This repository contains the official Javascript & Typescript SDK for the Ludex API.
+For usage and documentation head over to our [official documentation](https://docs.ludex.gg/ludex-sdks/javascript-sdk).
+
+## v1 Deprecation
+
+v1 is being deprecated as now we will be doing everything via api's from now on.
 
 ## Usage
 
-For usage and documentation head over to our [official documentation](https://docs.ludex.gg/ludex-sdks/javascript-sdk).
+#### Before You Begin
+
+Make sure you have the credentials for Ludex API Services.
+
+Organization Api key for organization scoped API's
+
+Client Api key for client scoped API's
+
+For further information on how to get api keys checkout our [docs](https://docs.ludex.gg/dashboard/get-your-api-keys)
+
+#### Requirements
+
+- [Node.js](https://nodejs.org) v16 or higher.
```

Comment:
I don't think we need that. The sdk is loaded on the frontend, node is not required

Reply:
Sdk is not loaded on frontend. Only backend